x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/planner/LocalExecutionPlanner.java

```diff
@@ -343,7 +343,8 @@ private PhysicalOperation planTopN(TopNExec topNExec, LocalExecutionPlannerConte
                 case "text", "keyword" -> TopNEncoder.UTF8;
                 case "version" -> TopNEncoder.VERSION;
                 case "boolean", "null", "byte", "short", "integer", "long", "double", "float", "half_float", "datetime", "date_period",
-                    "time_duration", "object", "nested", "scaled_float", "unsigned_long", "_doc" -> TopNEncoder.DEFAULT_SORTABLE;
+                    "time_duration", "object", "nested", "scaled_float", "unsigned_long", "_doc", "counter_integer", "counter_long",
+                    "counter_double" -> TopNEncoder.DEFAULT_SORTABLE;
```

Comment:
This'd allow us to sort on these. Do we want that?

Reply:
Definitely not, my bad. I fixed this in https://github.com/elastic/elasticsearch/pull/107877/commits/06b30783b80286edfbf80f48619ec7e0fca0e714.